editor/editor_themes.cpp

```diff
@@ -86,15 +86,15 @@ static Ref<StyleBoxFlat> change_border_color(Ref<StyleBoxFlat> p_style, Color p_
 	return style;
 }
 
-Ref<ImageTexture> editor_generate_icon(int p_index, bool dark_theme = true, Dictionary *p_colors = NULL) {
+Ref<ImageTexture> editor_generate_icon(int p_index, bool convert_color) {
```

Comment:
The convention is to use the `p_` prefix for parameters, so it should be `p_convert_color`.